content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -459,7 +459,7 @@ non-default field manager, as seen in the following example. The default field
 manager for kubectl server-side apply is `kubectl`.
```

Comment:
We might want to explain when you would want to set a different value than `kubectl`. Neither the existing nor the new doc make that clear.